src/pymodaq_plugins_cellkraft/daq_move_plugins/daq_move_CellkraftE1500.py

```diff
@@ -120,49 +113,31 @@ def get_actuator_value(self):
         -------
         float: The position obtained after scaling conversion.
         """
-        if self.current_axes == 'Flow':
-            flow = DataActuator(data=self.controller.Get_Flow())
-            flow = self.get_position_with_scaling(flow)
-            return flow
-
-        elif self.current_axes == 'RH':
-            air_h = DataActuator(data=self.controller.Get_Air_H())
-            air_h = self.get_position_with_scaling(air_h)
-            return air_h
-
-        elif self.current_axes == 'Steam_Temperature':
-            steam_t = DataActuator(data=self.controller.Get_Steam_T())
-            steam_t = self.get_position_with_scaling(steam_t)
-            return steam_t
-
-        elif self.current_axes == 'Tube_Temperature':
-            tube_t = DataActuator(data=self.controller.Get_Tube_T())
-            tube_t = self.get_position_with_scaling(tube_t)
-            return tube_t
-
-        elif self.current_axes == 'Pressure':
-            pressure = DataActuator(data=self.controller.Get_Pressure())
-            pressure = self.get_position_with_scaling(pressure)
-            return pressure
-
-        else:
-            self.emit_status(ThreadCommand('Update_Status',
+        match self.current_axes:
```

Comment:
If case is _, nothing is returned, is the code handling it ?

You can also simplify using the same variable for each case before scaling and returning it
```suggestion
        match self.current_axes:
            case 'Flow':
                value = DataActuator(data=self.controller.Get_Flow())
            case 'Steam_Temperature':
                value = DataActuator(data=self.controller.Get_Steam_T())
            case 'Tube_Temperature':
                value = DataActuator(data=self.controller.Get_Tube_T())
            case 'RH':
                value = DataActuator(data=self.controller.Get_Air_H())
            case 'Pressure':
                value = DataActuator(data=self.controller.Get_Pressure())
            case _:
                self.emit_status(ThreadCommand('Update_Status',
                ['WARNING - No Axis Selected, self.current_axes can be None']))
                value = DataActuator(data=0.)
        return value
```